Lib/asyncio/base_events.py

```diff
@@ -549,7 +551,7 @@ def _asyncgen_firstiter_hook(self, agen):
                     'asyncgen': agen
                 })
 
-    async def shutdown_default_executor(self):
+    async def shutdown_default_executor(self, timeout=THREAD_JOIN_TIMEOUT):
         """Schedule the shutdown of the default executor."""
```

Comment:
You'll need to update the docstring and documentation of this method & asyncio.run to reflect the change.

Reply:
Will do, just wanted to verify the implementation details first, I'll apply your suggestion and then update the docstring and docs accordingly.

Reply:
While updating the docstrings, I realized that the docstring for `asyncio.run()` wasn't updated to reflect the addition of `shutdown_default_executor()`. I'll address that as well, it was only a small change.